libs/design/src/scss/theming/_variables.scss

```diff
@@ -87,9 +87,22 @@ $daff-bronze: (
 	100: #1a0f00,
 );
 
+$daff-green: (
```

Comment:
Please modify the commit to include the `BREAKING CHANGE:` explaining why you broke the API.

Reply:
updated